example/algorithms_evaluation/fixed_points.py

```
@@ -0,0 +1,14 @@
+import sys
+
+from biodivine_aeon import *
+
+bn = BooleanNetwork.from_file(sys.argv[1])
+bn = bn.infer_valid_graph()
+
+stg = AsynchronousGraph(bn)
+
+fixed_points = FixedPoints.symbolic(stg)
```

Ruff (0.12.2)

`FixedPoints` may be undefined, or defined from star imports

(F405)